src/USER-MISC/angle_fourier_simple_approx.h

```diff
@@ -0,0 +1,93 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Tim Bernhard (ETHZ)
+------------------------------------------------------------------------- */
+
+#ifdef ANGLE_CLASS
+// clang-format off
+AngleStyle(fourier/simple/approx,AngleFourierSimpleApprox);
+// clang-format on
+#else
+
+#ifndef LMP_ANGLE_FOURIER_SIMPLE_APPROX_H
+#define LMP_ANGLE_FOURIER_SIMPLE_APPROX_H
+
+#include "angle_fourier_simple.h"
+#include "math_const.h"
+
+namespace LAMMPS_NS {
+
+/* ---------------------------------------------------------------------- */
+
+/**
+ * Approximation of cos
+ * maximum error is about 0.00109 for the range -pi to pi
+ * Source: https://stackoverflow.com/a/28050328/3909202
+ */
+static double fastCos(double x)
+{
+  constexpr double inv2pi = 1. / (MathConst::MY_2PI);
+  // TODO: check if range map is necessary
+  double x_wrapped = x - MathConst::MY_2PI * floor(x * inv2pi);
+  x_wrapped *= inv2pi;
+  x_wrapped -= 0.25 + floor(x_wrapped + 0.25);
+  x_wrapped *= 16.0 * (abs(x_wrapped) - 0.5);
```

Comment:
Shouldn't you be using fabs() here in both cases?

Reply:
You are right, thanks!